00 Boilerplate/README.md

```diff
@@ -3,10 +3,9 @@
 In this sample we are going to setup a web project that can be easily managed
 by webpack.
 
-We won't install anything related about Vue.js, just some basic plumbing.
+We won't install anything related to Vue.js, just some basic plumbing.
 
-We will setup an initial <abbr title="Node.js package manager, a package manager for the JavaScript runtime environment Node.js">npm</abbr> project, give support to TypeScript<br />
-Then we will create a **helloworld.ts** sample.
+We will setup an initial <abbr title="Node.js package manager, a package manager for the JavaScript runtime environment Node.js">npm</abbr> project and give support to TypeScript. Then we will create a **helloworld.ts** sample.
```

Comment:
Here I am just replacing a ' , ' by 'and'. I am not sure if this what you meant or maybe after the ' , ' you wanted to add more packages but you forgot.